frontend/views/utils/misc.js

```diff
@@ -23,3 +24,15 @@ export const showNavMixin = {
     }
   }
 }
+
+export const checkCypressMixin = {
+  computed: {
+    isInCypress (): boolean {
+      // NOTE: By only checking the window.Cypress, it's difficult to say that the app is running in Cypress
+      //       because anyone can access the window object.
+      //       in Cypress mode, sbp is exposed to window object (see main.js) so window.sbp can be compared
+      //       with our sbp object to verify if the app is in Cypress mode in real
+      return !!window.Cypress && window.sbp === sbp
+    }
+  }
+}
```

Comment:
Since anyone can access `window` object and set value in console, I think `!!window.Cypress` is not enough.
```
window.Cypress = true
```